asllib/doc/ASLTypingReference.tex

```diff
@@ -2697,7 +3920,10 @@ \section{TypingRule.SliceStar \label{sec:TypingRule.SliceStar}}
 \subsection{Comments}
     \identr{GXQG}: The notation b\texttt{i *: n} is syntactic sugar for b\texttt{i*n +: n}
```

Comment:
I fixed the syntactic sugar bitvector notations to include []

Reply:
done